galaxykubeman/templates/rbac.yaml

```diff
@@ -0,0 +1,30 @@
+kind: Role
```

Comment:
Maybe we should wrap the whole file in an `{{ if .Values.rbac.enabled }}` or some other key to choose whether to deploy these resources or not in case we ever want to use the chart to deploy other things without these... The default could be 
```
rbac:
  enabled: true
```
in values, so the default would be to deploy them, but we'd have the possibility to not deploy them through a quick values change.
I believe all other resources are currently optional and loop over keys so an empty dict for any of the other values would make those resources not deploy. We could do the same thing here if there's any chance we'd need to add more than just this one pair, but I don't see a reason for that in the forseeable future so a simple enabled on/off switch should do it and keep all resource deployment optional from the chart perspective

Reply:
that makes sense, I can update!